vos/docker/2.7/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM python:2.7-alpine
```

Comment:
I don't think offering these containers is a good idea. Python already has (and has had for a long time) its own virtualization environment. I understand that some people have a preference for containers but in the python world that has not become the mainstream (at least not yet). The main packages that I'm using do not have containers in general.

It's not a problem checking them in the repo, but once they are part of the repo, then we take a responsibility for ensuring that they work. We would need to create Travis tests and add them to the nightly builds. A lot of effort for little gain in IMO.

Reply:
OK.